### PR TITLE
fix variable name error from 'std'->'var'

### PR DESCRIPTION
**Pull request by qiuqiangkong**
According to the original Batch Normalization paper, the running average should be on 'var' not 'std'. In the keras implementation of BatchNormalization class, variable name 'std' is used wrongly and is equal to the value of variance. This is very confusing and if very difficult for readers to understand. Hence I corrected the name from 'std' to 'var'. The function and output of the BatchNormalization class keep unchanged. The code is throughly tested. 

Best wishes, 
Qiuqiang Kong

### CODE REVIEW

**Comment by fchollet** [comment]
This changes the public API (`self.running_var`) so we won't merge it. We'll fix the naming issue in Keras 2.
